src/neptune/management/exceptions.py

```diff
@@ -163,3 +163,8 @@ class BadRequestException(ManagementOperationFailure):
 class IncorrectIdentifierException(ManagementOperationFailure):
     code = 21
     description = "Can not parse '{identifier}' as identifier."
+
+
+class ExperimentNotFound(ManagementOperationFailure):
```

Comment:
I'm not sure about naming as we shouldn't be using `experiment` anywhere that is user-exposed. It's mostly legacy name. The most recent is probably MetadataContainer etc.